modules/common/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/sqlquery/SqlQueryEnrichmentIntegrationTest.scala

```diff
@@ -407,12 +405,7 @@ class SqlQueryEnrichmentIntegrationTest extends Specification {
     val config = SqlQueryEnrichment.parse(configuration, SCHEMA_KEY).map(_.enrichment)
     val context = config.toEither.flatMap(_.lookup(event, Nil, Nil, None).toEither)
 
-    context must beLeft.like {
-      case nel =>
-        nel.head.asInstanceOf[EnrichmentFailure].message must beEqualTo(
-          FailureDetails.EnrichmentFailureMessage.Simple("The placeholder map error. The map: Some(IntMap()), where count is: 1")
-        )
-    }
+    context ==== Right(Nil)
```

Comment:
What's the difference with `beRight(Nil)`?

Reply:
The `====` is safe in term of comparing different types - it doesn't compile. The `must beRight` isn't.